Swish/Sources/ProjectLib/Project.swift

```diff
@@ -1,8 +1,60 @@
 import Sh
 
 public struct Project {
-  public init() {}
-  public func generate() throws {
-    try sh(.terminal, "xcodegen")
-  }
-}
\ No newline at end of file
+    let openBytesInitialHeader = """
+                                                <?xml version="1.0" encoding="UTF-8"?>
+                                                <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+                                                <plist version="1.0">
+                                                <dict>
+                                                    <key>FILEHEADER</key>
+                                                    <string>
+                                                //  ___PROJECTNAME___ template generated by OpenBytes on ___DATE___.
+                                                //  Copyright (c) ___YEAR___ OpenBytes
+                                                //
+                                                //  Permission is hereby granted, free of charge, to any person obtaining a copy
+                                                //  of this software and associated documentation files (the "Software"), to deal
+                                                //  in the Software without restriction, including without limitation the rights
+                                                //  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+                                                //  copies of the Software, and to permit persons to whom the Software is
+                                                //  furnished to do so, subject to the conditions found at the following link:
+                                                //  https://github.com/0xOpenBytes/ios-base/blob/main/LICENSE
```

Comment:
This will be updated

Reply:
Fixed in https://github.com/0xOpenBytes/ios-base/pull/20/commits/f8267216f8e5232095f37c7a373476af89bf07f9